src/ice_model.F90

```diff
@@ -2588,7 +2594,11 @@ subroutine ice_model_init(Ice, Time_Init, Time, Time_step_fast, Time_step_slow,
   else
     Ice%xtype = REDIST
   endif
-
+  
+  if (fast_ice_PE .eqv. slow_ice_PE) then
```

Comment:
Should the `.eqv.` here be an `.and.`?

Reply:
This has been changed to .and. in the updated version.